composer.lock

```diff
@@ -11304,5 +12314,5 @@
         "lib-libxml": "*"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.0.0"
+    "plugin-api-version": "1.1.0"
```

Comment:
Please generate composer.lock using Composer version 2 to minimize changes

Reply:
Hello @sivaschenko 
Thank you for notice.
It has been fixed.